.travis.yml

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2013-2020 Arm Limited. All rights reserved.
```

Comment:
Shouldn't this be just:
```suggestion
# Copyright (c) 2020 Arm Limited. All rights reserved.
```
?

If so, you may need to update every example projects you added this file.